xbmc/cores/VideoPlayer/VideoRenderers/WinRenderer.h

```diff
@@ -235,7 +235,7 @@ class CWinRenderer : public CBaseRenderer
   unsigned int         m_destHeight;
 
   int                  m_neededBuffers;
-  unsigned int         m_frameIdx;
+  unsigned int         m_frameIdx = 0;
```

Comment:
in any case i don't think that this may cause any issue, it's used only in case of DXVA rendering.